Messaging.md

```diff
@@ -399,14 +421,10 @@ See [the Oracle specifications](./Oracle.md#oracle-attestations) for more detail
 1. type: 55400 (`oracle_attestation_v0`)
 2. data:
     * [`string`:`event_id`]
-    * [`x_point`:`oracle_public_key`]
+    * [`x_point`:`oracle_attestation_public_key`]
```

Comment:
Similarly would be good to mention somewhere in text that oracle needs different keys.